exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook/data/data_customer_support.ipynb

```diff
@@ -0,0 +1,147 @@
+{
```

Comment:
<p>I don't know if this is a helpful paragraph. As we discussed, the results may look a bit weird. Perhaps we do want to say something in our defence - explain that the data is not real. The real one on this subject is hard to get, etc. But saying that it is a good representation of a real world data is like admitting that the whole thing has a very limited practical use.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/exasol/ai-lab/pull/364/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>